styles/adoptadog-stylesheet.css

```diff
@@ -0,0 +1,154 @@
+header {
```

Comment:
You might consider adding 
`*{margin:0;}`
at the top of the style sheet. This trick removes any default browser styles and only applies margins explicitly set by your css. For me, Chrome always adds a margin to the body by default, ruining some of the header and footer styles.